.travis.yml

```diff
@@ -24,6 +24,8 @@ install:
 
 script:
   - poetry run py.test tests/
+  - poetry run mypy rsa/
```

Comment:
Just found that `mypy` is already running in CI as a test case in `pytest`.

Do I need to add `flake8` as a test also or can I just drop the `mypy` test?

Reply:
Please also add a flake8 test. That way it's possible to just run `py.test` and have everything tested in one go.

Reply:
I would like to make it in a separate PR to keep it focused. I will revert my current `.travis.yml` changes.